ApiBundle/Controller/ContentController.php

```diff
@@ -150,6 +151,40 @@ public function deleteContentsAction(Request $request)
     }
 
     /**
+     * @param Request $request
+     * @param string  $contentId
+     * @param string  $language
+     *
+     * @Config\Route("/delete-multiple-version/{contentId}/{language}", name="open_orchestra_api_content_delete_multiple_versions")
+     * @Config\Method({"DELETE"})
+     *
+     * @return Response
+     */
+    public function deleteContentVersionsAction(Request $request, $contentId, $language)
+    {
+        $format = $request->get('_format', 'json');
+        $facade = $this->get('jms_serializer')->deserialize(
+            $request->getContent(),
+            $this->getParameter('open_orchestra_api.facade.content_collection.class'),
+            $format
+        );
+        $contents = $this->get('open_orchestra_api.transformer_manager')->get('content_collection')->reverseTransform($facade);
+        $nbrVersions = $this->get('open_orchestra_model.repository.content')->countNotDeletedByLanguage($contentId, $language);
```

Comment:
@alavieille : We should only use explicit variable names, with no abbreviation : maybe versionsCount instead of nbrVersions